src/kvstore/raftex/Host.h

```diff
@@ -132,6 +127,8 @@ class Host final : public std::enable_shared_from_this<Host> {
 
     LogID committedLogId_{0};
     std::atomic_bool sendingSnapshot_{false};
+
+    thrift::ThriftClientManager<cpp2::RaftexServiceAsyncClient> connManager_;
```

Comment:
We should share the same instance of ThriftClientManager between different hosts.

Reply:
We may need a better way to fix this. So I will close this PR first.